### PR TITLE
CG delivery report

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
Testable by running CG report delivery 'cust000' 'happytapir' > 'happytapir.html'

### CODE REVIEW

File: cg/apps/lims/api.py [hunk 1]
```diff
@@ -12,6 +12,17 @@
...
+    '1464': 'Automated TruSeq DNA PCR-free library preparation method',
+    '1317': 'HiSeq X Sequencing method at Clinical Genomics',
+    '1383': 'MIP analysis for Whole genome and Exome',
+    '1717': 'NxSeq® AmpFREE Low DNA Library Kit (Lucigen)',
+    '1060': 'Raw data delivery',
+    '1036': 'HiSeq 2500 Rapid Run sequencing',
+    '1314': 'Automated SureSelect XT Target Enrichment for Illumina sequencing',
+    '1518': '200 ng input Manual SureSelect XT Target Enrichment',
+    '1079': 'Manuel SureSelect XT Target Enrichment for Illumina sequencing',
+}
```
Comment (ingkebil) [inline cg/apps/lims/api.py]: Are we only getting the version number from LIMS?

Comment (ingkebil) [reply cg/apps/lims/api.py]: If we are getting document_id, document_version and document_title from LIMS, we don't need these constants anymore, right?

Comment (patrikgrenfeldt) [reply cg/apps/lims/api.py]: The document_title is not in LIMS but I created an issue about introducing this information into the CG database

**Comment by patrikgrenfeldt** [comment]
nice code. we cannot push it yet tho :) Depends on MIP6.0 /KB

**Pull request merged**